waku/waku_filter_v2/protocol.nim

```diff
@@ -156,27 +156,28 @@ proc pushToPeer(wf: WakuFilter, peer: PeerId, buffer: seq[byte]) {.async.} =
 
   if not wf.peerManager.peerStore.hasPeer(peer, WakuFilterPushCodec):
     # Check that peer has not been removed from peer store
-    trace "no addresses for peer", peer = peer
+    error "no addresses for peer", peer_id = shortLog(peer)
     return
 
   ## TODO: Check if dial is necessary always???
   let conn = await wf.peerManager.dialPeer(peer, WakuFilterPushCodec)
   if conn.isNone():
     ## We do not remove this peer, but allow the underlying peer manager
     ## to do so if it is deemed necessary
-    trace "no connection to peer", peer = peer
+    error "no connection to peer", peer_id = shortLog(peer)
     return
 
   await conn.get().writeLp(buffer)
 
 proc pushToPeers(
     wf: WakuFilter, peers: seq[PeerId], messagePush: MessagePush
 ) {.async.} =
-  debug "pushing message to subscribed peers",
+  info "pushing message to subscribed peers",
     pubsubTopic = messagePush.pubsubTopic,
     contentTopic = messagePush.wakuMessage.contentTopic,
-    peers = peers,
-    hash = messagePush.pubsubTopic.computeMessageHash(messagePush.wakuMessage).to0xHex()
+    target_peer_ids = peers.mapIt(shortLog(it)),
+    msg_hash =
+      messagePush.pubsubTopic.computeMessageHash(messagePush.wakuMessage).to0xHex()
```

Comment:
It might be called twice per logging. There is a processing flow in chronicles that evaluates twice the message arguments. It is better as its in line 214 in this file.